superset-frontend/src/components/Select/utils.tsx

```diff
@@ -16,33 +16,22 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { ensureIsArray, styled, t } from '@superset-ui/core';
-import { Spin } from 'antd';
-import Icons from 'src/components/Icons';
-import AntdSelect, {
-  SelectProps as AntdSelectProps,
-  SelectValue as AntdSelectValue,
-  LabeledValue as AntdLabeledValue,
-} from 'antd/lib/select';
+import { ensureIsArray, t } from '@superset-ui/core';
+import AntdSelect, { LabeledValue as AntdLabeledValue } from 'antd/lib/select';
 import { rankedSearchCompare } from 'src/utils/rankedSearchCompare';
-import {
-  OptionTypeBase,
-  ValueType,
-  OptionsType,
-  GroupedOptionsType,
-} from 'react-select';
-import React, {
-  ReactElement,
-  ReactNode,
-  RefObject,
-  JSXElementConstructor,
-} from 'react';
+import React, { ReactElement, RefObject } from 'react';
 import { DownOutlined, SearchOutlined } from '@ant-design/icons';
-
-declare type RawValue = string | number;
+import { StyledHelperText, StyledLoadingText, StyledSpin } from './styles';
+import { LabeledValue, RawValue, SelectOptionsType, V } from './types';
 
 const { Option } = AntdSelect;
 
+export const MAX_TAG_COUNT = 4;
+
+export const TOKEN_SEPARATORS = [',', '\n', '\t', ';'];
+
+export const EMPTY_OPTIONS: SelectOptionsType = [];
```

Comment:
Should these (and others) be placed in a `constants.ts`?

Reply:
Great idea